app/app.go

```diff
@@ -142,6 +143,7 @@ func getGovProposalHandlers() []govclient.ProposalHandler {
 		upgradeclient.CancelProposalHandler,
 		ibcclientclient.UpdateClientProposalHandler,
 		ibcclientclient.UpgradeProposalHandler,
+		stakeibcclient.AddValidatorProposalHandler,
```

Comment:
Should we add a GovProposal to change validator weight? 

This isn't necessarily urgent, and we can easily structure it in a way where this is done through signaling proposals for now, and then the foundation takes care of executing the messages. But, it is something to think about imo

Reply:
resolved offline